### PR TITLE
feat(draw.io): add presence

### PR DESCRIPTION
**Pull request by HopperElec**
## Description 
Add presence for draw.io (AKA diagrams.net)

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image
image
image

</details>

### CODE REVIEW

File: websites/D/draw.io/presence.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	};
+	if (document.title === "draw.io")
+		presenceData.details = "Creating a new diagram";
+	else {
+		presenceData.details = "Editing a diagram";
+		if (!(await presence.getSetting<boolean>("privacy")))
+			presenceData.state = document.title.replace(/ - draw.io$/, "");
+	}
+	presence.setActivity(presenceData);
+});
```
Comment (DarkVillager) [inline websites/D/draw.io/presence.ts:18]: Low quality presence.

Comment (theusaf) [reply websites/D/draw.io/presence.ts:18]: @HopperElec Is there anything else you can add to this presence?

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/D/draw.io/presence.ts (hunk 1 above)
Comment (HopperElec) [reply websites/D/draw.io/presence.ts:18]: The draw.io site does not have a home page or anything- visiting it immediately prompts you to open or create a new diagram. They have [a separate site](https://www.drawio.com/) but this seems to be aimed more towards businesses considering using them and, especially when you consider that a PreMiD user who installs this presence would already be using it, it seems unnecessary to include this site in the presence. However, if you disagree then I'd be happy to implement it.

Comment (theusaf) [reply websites/D/draw.io/presence.ts:18]: Got it.

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**